### ISSUE TITLE
FEN - in progress not updating

### ISSUE THREAD

**Issue by cojoMan** [Tikipeter/repository.tikipeter#756]
Was wondering if anyone else has this issue - worked just fine up until..a week or so ago.

- Trakt authenticated, as a Kodi program addon, and inside FEN
- using "movies watchlist" to start smth up
- it used to be that is I went to Movies/Progress, the last started/scrobbled movie would pop in here, at the top or the bottom of the list. Currently I only get the Watchlist. Cleared cache, reauthenticated FEN/trakt, same.

I couldn't find an "updated setting/option" that would explain this behaviour since I assume it has happened after the last update. Thank you.

**Comment by Tikipeter** [comment]
I'm sure what you've written above makes perfect sense to you, but it doesn't really to me.

Are you saying that when you navigate into Movies->In Progress, that you instead see all the movies in your Trakt Watchlist?

**Comment by cojoMan** [comment]
You are correct. And I apologize for my messy description. Hope I can do better.

I just rechecked before replying, and a few notes :
- as per above - the current results of my "Movies/In Progress" shows me a list of partially started movies that looks a lot like my Watchlist even though not 1:1. Seems like a lot of "in progress" movies, but started too long ago to remember starting any of them.
- as a test I've started a few random movies, wait for the scrobble to start, stopped the movie. Those do not appear in Movies/In Progress. Clearing the cache via Fen/Utilities did nothing as well.

Could it be that Trakt is somewhat down/messed up right now ? Cannot find a Trakt section for in progress movies to check though. Will try installing a few other addons and check those, maybe reinstall FEN.

**Comment by cojoMan** [comment]
Oh, and of course, I did check the left side panel options, the filter for "watched/unwatched/all videos" is set to "all videos", wasn't that...

**Comment by ShibajiCh** [comment]
Hi @cojoMan,

The following tool lets you see what you have in your Trakt In Progress list:

https://sharkykh.github.io/tppm/

Please login with your Trakt ID and check if these entries matches the ones with the Fen's In Progress list.

For me the Fen In Progress list is working fine.

Thanks and Regards,
ShibajiCh.

**Comment by cojoMan** [comment]
The lists do not match for me. I'll try a clean install in a sandbox, but so far, on my PC and my nvidiaShield I get the same behaviour...

**Comment by Tikipeter** [comment]
No point installing in a sandbox.

Just post me a log first and I'll see if there's anything in there.

**Comment by cojoMan** [comment]
https://paste.kodi.tv/jibevicebo

do I need to do anything specific before going "Fen/Tools/Changelog and Log Utils/Upload Kodi Log to Pastebin" ?

**Comment by Tikipeter** [comment]
>Trakt authenticated, as a Kodi program addon, and inside FEN

I just re-read your original post, and saw you'd written this. Deciphering this I think it means you have the Trakt addon from the official repo installed. If so, you need to go into it's settings and make sure it is ignoring media being played from an internet stream (i.e. from an addon). I don't have the Trakt addon installed so can't remember the setting. If you haven't done this, then Fen will see that you have that addon installed and not set up to ignore internet streams, and will not do any marking as watched/progress.

As an aside, you need to think about whether you need the Trakt addon installed at all. It is really for people with Kodi libraries. It is almost always not needed if you are using Trakt through an addon, as the addon needs to be the one handling the Trakt interactions.

**Comment by cojoMan** [comment]
"Exclude Plugins", found it. Was disabled, will try enabling it and see what happens. Alternatively will try to disable it entirely.

I am not using a kodi Library, but I do send out various http streams to kodi via a remote app, and that helps scrobbling, or so I thought.

Will make a few more test. Thank you for taking some time to look at this ! :)

**Comment by cojoMan** [comment]
so, a few things :
- I've disabled the trakt program addon, started a movie, stopped it, refreshed the TTMP app, I can see progress there. Started the movie again from FEN, resumed where it should. Will see if I need the separate Trakt addon down the line at some point. Seems to be behaving well so far.
- FEN - Navigation, sorting was all set to TITLE. I've changed some of those to "Recently Watched" and everything seems to behave as before. Started a few movies I never did before as a test, and they pop up at the top. What I was getting before and thought was "Watchlist" was in fact the whole of my "In progress", alphabetically sorted, so it looked quite a lot the same. Apologies.

Not sure how those flags got reset, I don't remember messing with them, but the behaviour of "progress" changed for sure.
Thank you for all the help and what I've learned - eg. not needing the trakt plugin (probably) and the TPPM app :)
